guides/express-weather-mcp-server/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "express-weather-mcp-server",
+  "repository": "https://github.com/vercel/examples.git",
+  "license": "MIT",
+  "type": "module",
+  "private": true,
+  "scripts": {
+    "dev": "tsx watch src/index.ts",
+    "build": "tsc",
+    "start": "node dist/index.js"
```

Comment:
```suggestion
```
Should just omit these, `vc build` will handle, we're working on improving this story, but for now this is our recommendation.